star-rx72n-firmware/src/rx_clock_power_init.c

```
@@ -156,47 +172,105 @@ static rx_err_t internal_clock_init(void)
  *
  * Note: SCI modules are enabled per-channel in uart_init_channel()
  *
- * @return k_rx_ok on success
+ * @return k_rx_ok on success, k_rx_err if verification fails after retries
  */
 static rx_err_t internal_module_stop_init(void)
 {
-  /* Protect off */
-  system_regs()->prcr = k_rx_prcr_unlock_all;
+  const uint32_t mstpcra_clear_mask = (1UL << k_mstpcra_cmt) | (1UL << k_mstpcra_mtu);
```

Clang (14.0.6)

[warning] variable 'mstpcra_clear_mask' is not initialized

(cppcoreguidelines-init-variables)